webdriver-spec.html

```diff
@@ -2850,12 +2853,15 @@
   or for the <a>session page load timeout</a> milliseconds to pass,
   whichever occurs sooner.
 
-  <p class=issue>This doesn’t actually wait for the page to load
-   if traversing the session history kicks off a navigation.
+  <li><p>If traversing the session history causes <a>navigation</a>:
```

Comment:
Well if this happens post every command, you don’t need to say it specifically for backwards and forwards.